fdbserver/ClusterController.actor.cpp

```diff
@@ -3959,7 +3957,9 @@ ACTOR Future<Void> timeKeeper(ClusterControllerData* self) {
 				Version v = tr->getReadVersion().get();
 				int64_t currentTime = (int64_t)now();
 				versionMap.set(tr, currentTime, v);
-
+				if (!g_network->isSimulated()) {
+					TraceEvent("TimeKeeperCommit", debugID).detail("Version", v).log();
```

Comment:
You can put the `.log()` at the end, but it's not necessary for trace events like this where we aren't storing them in any local variable. It will be immediately destructed and that calls `log` for you.

```suggestion
					TraceEvent("TimeKeeperCommit", debugID).detail("Version", v);
```